Cargo.toml

```diff
@@ -8,6 +8,7 @@ homepage    = "https://github.com/tendermint/kms/"
 readme      = "README.md"
 categories  = ["cryptography"]
 keywords    = ["cosmos", "ed25519", "kms", "key-management", "yubihsm"]
+edition     = "2018"
```

Comment:
Guess we never actually completed the 2018 edition upgrade.

Reply:
Huh, all the 2018 changes in #138 and we did not actually put in that `edition = "2018"` line 🤦‍♂️